tests/AtlasConnectivity.Tests/AtlasConnectivity.Tests.csproj

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The Atlas connectivity tests are packaged in a separate DLL because a DLL is the only unit at which it is easy to select which tests to run and we want to run regular tests separately from Atlas connectivity tests.